src/components/Practice/AserFlow.jsx

```
@@ -1,61 +1,39 @@
-import hintimg from "../../assets/hintsicon.svg";
-import { MessageDialog } from "../Assesment/Assesment";
+import { callTelemetryDiscovery } from "../../utils/apiUtil";
+import {
+  getLocalData,
+  ListenButton,
+  sendTestRigScore,
+  setLocalData,
+} from "../../utils/constants";
 import { DISCOVERY_SET_FLOW_STORAGE } from "../../utils/discoverSetFlow";
+import { nextimg as nextImg } from "../../utils/imageAudioLinks";
+import { MessageDialog } from "../Assesment/Assesment";
```

ESLint

[error] Unable to resolve path to module '../Assesment/Assesment'.

(import/no-unresolved)